container/lxd/initialisation_test.go

```diff
@@ -327,7 +333,7 @@ func (s *InitialiserSuite) TestFindAvailableSubnetUpperBoundAndLowerBoundInUse(c
 	})
 	subnet, err := findNextAvailableIPv4Subnet()
 	c.Assert(err, gc.IsNil)
-	c.Assert(subnet, gc.Equals, "1")
+	c.Assert(subnet, gc.Equals, "4")
 }
```

Comment:
It might also be nice for Juju to be able to pass in other known subnets. Whether they are Provider subnets or other LXD subnets we know about.

If it's hard I wouldn't do too much, as I don't want to spend too much time on this.